docs/source/admin/traffic_ops.rst

```diff
@@ -477,6 +477,12 @@ This file deals with the configuration parameters of running Traffic Ops itself.
 	.. versionadded:: 5.0
 		This is an optional boolean value to enable the handling of the "If-Modified-Since" HTTP request header. Default: false
 
+:role_based_permissions: Toggle whether or not to use Role Based Permissions.
+
+	.. versionadded:: 7.0
+		The blueprint can be seen :pr:`{{5848}}`
```

Comment:
I just wrapped it with `{{` and `}}` to show that it was a value to be replaced. This should just be
```rst
:pr:`5848`
```